bit/prjtrellis/meta.yaml

```diff
@@ -1,7 +1,9 @@
 # Use `conda-build-prepare` before building for a better version string.
 {% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v', '') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
 # cmake 1.15 required for boost 1.71
```

Comment:
Remove this comment; it described why boost 1.71 couldn't have been used.

Reply:
I've fixed it